cmd/openshift/operator/kodata/tekton-pipeline/0.24.3/00-pipelines.yaml

```diff
@@ -0,0 +1 @@
+./../../../../../../upstream/pipelines/0.24.3/release.yaml
```

Comment:
So this would be at the root of the git repository right ? I think we should / could use something a bit different, like `payload` or something ? (`upstream` doesn't speak too much to me 😛 )

Reply:
`payload` is fine 😄